src/commands/config.command.ts

```diff
@@ -0,0 +1,12 @@
+import {batchImportExportService} from "../services/package-manager/batch-import-export-service";
+
+export class ConfigCommand {
+
+    public async listActivePackages(jsonResponse: boolean, flavors: string[], withDependencies: boolean, packageKeys:string[]): Promise<void> {
+        if (jsonResponse) {
+            await batchImportExportService.findAndExportListOfActivePackages(flavors ?? [], packageKeys ?? [], withDependencies)
+        } else {
+            await batchImportExportService.listActivePackages(flavors ?? []);
+        }
+    }
```

Comment:
super-nit:
```suggestion
    public listActivePackages(jsonResponse: boolean, flavors: string[], withDependencies: boolean, packageKeys:string[]): Promise<void> {
    return jsonResponse 
      ? batchImportExportService.findAndExportListOfActivePackages(flavors ?? [], packageKeys ?? [], withDependencies)
      : batchImportExportService.listActivePackages(flavors ?? []);
    }
```

Reply:
I think its more readable with the if/else structure and also used in that way in other commands that have the json option as well.